### PR TITLE
test: add test file with special chars (DEV-4098)

### PR DESCRIPTION
**Pull request by jnussbaum**

### CODE REVIEW

File: src/dsp_tools/commands/xmlupload/models/ingest.py [hunk 1]
```diff
@@ -94,7 +93,7 @@ def _ingest(self, filepath: Path) -> IngestResponse:
         Returns:
             IngestResponse: The internal filename of the uploaded file.
         """
-        filename = urllib.parse.quote(unicodedata.normalize("NFKD", filepath.name))
+        filename = urllib.parse.quote(filepath.name)
```
Comment (jnussbaum) [inline src/dsp_tools/commands/xmlupload/models/ingest.py]: @siers : Previously, I had tried out to do an NFKD normalization of files with special characters, before constructing the ingest URL. But now, I see that it also works without. What do you think is better / more future-proof: Doing it or not? 

(I'm refering to the POST /projects/<shortcode>/assets/ingest/filename route)

File: src/dsp_tools/resources/start-stack/docker-compose.yml [hunk 2]
```diff
@@ -62,7 +62,7 @@ services:
   ingest:
     # on the verge of every deployment (fortnightly), update the "image" value from the "ingest" value of
     # https://github.com/dasch-swiss/ops-deploy/blob/main/versions/RELEASE.json
-    image: daschswiss/dsp-ingest:v0.12.2
+    image: daschswiss/dsp-ingest:v0.12.2-7-gf5cfb90
```
Comment (jnussbaum) [inline src/dsp_tools/resources/start-stack/docker-compose.yml:65]: Only merge this PR after the next deployment.

File: src/dsp_tools/commands/xmlupload/models/ingest.py (hunk 1 above)
Comment (siers) [reply src/dsp_tools/commands/xmlupload/models/ingest.py]: Since it is now normalized in the backend, it is not strictly necessary.

Comment (jnussbaum) [reply src/dsp_tools/commands/xmlupload/models/ingest.py]: Thanks, then I won't do it.

**Review by Nora-Olivia-Ammann — COMMENTED** [review]
While you're at it could you include all the german and french special characters?

**Review by Nora-Olivia-Ammann — APPROVED** [review]

**Review by siers — APPROVED** [review]

**Pull request merged**